app/controllers/ScriptController.scala

```diff
@@ -52,11 +53,12 @@ class ScriptsController @Inject()(val messagesApi: MessagesApi) extends Controll
   def update(scriptId: String) = SecuredAction.async(parse.json) { implicit request =>
     withJsonBodyUsing(scriptPublicReads) { scriptFromForm =>
       for {
-        oldScript <- ScriptDAO.findOneById(scriptId) ?~> Messages("script.notFound")
+        scriptIdValidated <- ObjectId.parse(scriptId)
+        oldScript <- ScriptSQLDAO.findOne(scriptIdValidated) ?~> Messages("script.notFound")
```

Comment:
Not sure if it's worth to refactor those two lines to a function. It's used 4 times.